### DISCUSSION TITLE
Fresh install on Debian 11 is giving HTTP 500 errors

### DISCUSSION THREAD

**Discussion by fongaboo** [postfixadmin/postfixadmin#682] [Q&A]
The initial login and main interface seems to be behaving. But if I do either of the following, I get an HTTP ERROR 500:

- OVERVIEW --> <domain>
- ADD FORWARD --> ADD FORWARD

I'm not sure that I can isolate it to MySQL queries because I can create admins and perform other functions that would definitely hit the database.

When I perform either of the above actions, I get the following in my Apache logs:

`errorSSL.log:[Thu Nov 10 19:51:19.625060 2022] [php7:error] [pid 25434] [client <my-home-ip-address>:49509] PHP Fatal error:  Uncaught  --> Smarty Compiler: Syntax error in template "file:/opt/postfixadmin/templates/list-virtual_mailbox.tpl"  on line 38 "{PALANG.To_Mailbox}<br/>"  - Unexpected ".", expected one of: "}" <-- \n  thrown in /opt/postfixadmin/vendor/smarty/smarty/libs/sysplugins/smarty_internal_templatecompilerbase.php on line 38, referer: https://<server-hostname>/postfixadmin/edit.php?table=alias`

I get nothing in the PHP logs.

If I check out line 38 of /opt/postfixadmin/templates/list-virtual_mailbox.tpl, I see nothing the jumps out at me as being wrong:

```
{if $display_mailbox_aliases==true}
                                <td>
                                {if $item.goto_mailbox == 1}
                                        {PALANG.To_Mailbox}<br/>
                                {else}
----->                               {PALANG.To_Forward_Only}<br/>
                                {/if}
                                {foreach from=$item.goto_other item=item2 key=j}
                                        {if $search eq ""}
                                                {$item2}
                                        {else}
                                                {$item2|replace:$search:"<span class='searchresult'>$search</span>"}
                                        {/if}
                                        <br/>
                                {/foreach}
                                </td>
                        {/if}

```

Then if I check line 38 of /opt/postfixadmin/vendor/smarty/smarty/libs/sysplugins/smarty_internal_templatecompilerbase.php, I see:

```
    /**
     * Smarty object
     *
     * @var Smarty
     */
    public $smarty = null;
```

Not much indication there either. Not having much luck with GoogleFu.

Any help would be appreciated!

**Comment by DavidGoodwin** [comment]
Hello - as I answered on sourceforge (although I prefer using github, so thanks for duplicating here) -

Could you please confirm what version of PostfixAdmin and PHP you are using?
